### PR TITLE
Save network and transmission matrix

### PR DESCRIPTION
**Pull request by andsv2**
Issue: "tergmLite replaces save.network and save.transmat. If FALSE, then save these two things. If TRUE, don't save them."

### CODE REVIEW

File: R/saveout.R [hunk 1]
```diff
@@ -142,18 +142,21 @@ saveout.net <- function(dat, s, out = NULL) {
...
     }
-    if (dat$control$save.transmat == TRUE) {
-      if (!is.null(dat$stats$transmat)) {
-        row.names(dat$stats$transmat) <- 1:nrow(dat$stats$transmat)
-        out$stats$transmat <- list(dat$stats$transmat)
-      } else {
-        out$stats$transmat <- list(data.frame())
+
+    if (dat$control$tergmLite != TRUE) {
+      if (dat$control$save.transmat == TRUE) {
```
Comment (smjenness) [inline R/saveout.R:147]: There should not be a `save.transmat` parameter at all anymore. tergmLite replaces that

File: R/saveout.R [hunk 2]
```diff
@@ -142,18 +142,21 @@ saveout.net <- function(dat, s, out = NULL) {
...
+          out$stats$transmat <- list(data.frame())
+        }
+        class(out$stats$transmat) <- c("transmat", class(out$stats$transmat))
       }
-      class(out$stats$transmat) <- c("transmat", class(out$stats$transmat))
-    }
 
-    if (dat$control$save.network == TRUE) {
-      out$network <- list(dat$nw)
+      if (dat$control$save.network == TRUE) {
```
Comment (smjenness) [inline R/saveout.R:157]: Same here

File: tests/testthat/test-net-long.R [hunk 3]
```diff
@@ -3,7 +3,7 @@ context("Network extended models")
 ################################################################################
 
 test_that("edges models", {
-  skip_on_cran()
+  #skip_on_cran()
```
Comment (smjenness) [inline tests/testthat/test-net-long.R:6]: Put this back to uncommented. To run these blocks locally, you can use:
`Sys.setenv("NOT_CRAN"="true")`

**Comment by smjenness** [comment]
Please review and edit before merging.

File: R/saveout.R (hunk 2 above)
Comment (andsv2) [reply R/saveout.R]: Input parameters for save.network/transmat have been removed from EpiModel with tergmLite handling their previous usage.

File: R/saveout.R [hunk 4]
```diff
@@ -142,18 +142,18 @@ saveout.net <- function(dat, s, out = NULL) {
     if (dat$control$save.nwstats == TRUE) {
       out$stats$nwstats <- list(dat$stats$nwstats)
     }
-    if (dat$control$save.transmat == TRUE) {
+
+    if (dat$control$tergmLite == FALSE) {
       if (!is.null(dat$stats$transmat)) {
         row.names(dat$stats$transmat) <- 1:nrow(dat$stats$transmat)
```
[the remark this answers is not in the document]
Comment (andsv2) [reply R/saveout.R:148]: @smjenness running through some of the tests: in ```test-merge.R``` there is a high infection rate and one initial infected in a pop'l of 100. In the edge list, the infected is not present in any partnerships which leads to a null del and subsequent null transmat. This causes saveout.R to error. Is that expected behavior?

File: R/net.utils.R [hunk 5]
```diff
@@ -142,8 +142,8 @@ check_bip_degdist <- function(num.g1, num.g2,
 #'
 #' Using the \code{color_tea} function with a \code{netsim} object requires that
 #' TEAs for disease status be used and that the \code{networkDynamic} object be
-#' saved in the output: both \code{tergmLite} and \code{save.network} must be
-#' set to \code{TRUE} in \code{\link{control.net}}.
+#' saved in the output: \code{tea.status} must be  set to \code{FALSE} in
+#' \code{\link{control.net}}.
```
[the remark this answers is not in the document]
Comment (andsv2) [reply R/net.utils.R:146]: Removed ```tea.status``` from ```color_tea```. Looks like this was the only remnant.

**Comment by andsv2** [comment]
@smjenness: review changes made. Passing on release and dev.

**Pull request merged**